src/main/java/com/listywave/user/repository/user/elastic/UserElasticRepository.java

```diff
@@ -0,0 +1,114 @@
+package com.listywave.user.repository.user.elastic;
+
+import static com.listywave.common.exception.ErrorCode.ELASTICSEARCH_REQUEST_FAILED;
+import static com.listywave.common.util.PaginationUtils.checkEndPage;
+
+import co.elastic.clients.elasticsearch.ElasticsearchClient;
+import co.elastic.clients.elasticsearch._types.query_dsl.MatchQuery;
+import co.elastic.clients.elasticsearch._types.query_dsl.Query;
+import co.elastic.clients.elasticsearch._types.query_dsl.TermQuery;
+import co.elastic.clients.elasticsearch.core.SearchRequest;
+import co.elastic.clients.elasticsearch.core.SearchResponse;
+import com.listywave.common.exception.CustomException;
+import com.listywave.user.application.domain.UserDocument;
+import com.listywave.user.application.dto.search.UserElasticSearchResult;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@RequiredArgsConstructor
+public class UserElasticRepository {
+
+    private final ElasticsearchClient esClient;
+
+    public Map<String, Object> findAllByElasticSearch(String search, Pageable pageable, Long loginUserId) {
```

Comment:
Service 계층과 파라미터 순서를 통일시키는 건 어떤가요??

```suggestion
    public Map<String, Object> findAllByElasticSearch(Long loginUserId, String search, Pageable pageable) {
```